src/main/java/no/digipost/api/client/representations/sender/SenderFeatureName.java

```diff
@@ -39,6 +39,16 @@ public final class SenderFeatureName {
      */
     public static final SenderFeatureName DIGIPOST_DELIVERY = new SenderFeatureName("no.digipost.feature.delivery.digipost", false);
 
+    /**
+     * Behandler kan sende masseutsendelser.
+     */
+    public static final SenderFeatureName MASSEUTSENDELSE = new SenderFeatureName("no.digipost.feature.batch.masseutsendelse", false);
+
+    /**
+     * Behandler kan sende masseutsendelser.
```

Comment:
Kopieringsglipp? "Behandler kan be om mottakersplitt"?

Reply:
Force pusher fiks. Godt øye!